<launch>
    <node pkg="gnn_tracker" type="gnn_tracker_node" name="gnn_tracker_node" output="screen">
        
        <!-- General parameters -->
        <param name="subs_topic" type="string" value="lrrObjects" />
        <param name="marker_pub_topic" type="string" value="meas_markers" />
        <param name="rate" type="int" value="50" />
        <param name="input_length" type="int" value="14" />

        <!--  Algorithm params -->
        <param name="proc_var" type="double" value="15.0" />
        <param name="meas_var" type="double" value="3.0" />
        <param name="detect_prob" type="double" value="0.95" />
        <param name="gate_threshold" type="double" value="1" />
        
    </node>
     <!--  -->
    <node type="rviz" name="rviz" pkg="rviz" args="-d ~/ros_ws/src/mtt_ros/gnn_tracker/rviz/markers.rviz" />
</launch>